### ISSUE TITLE
Include tcgplayer.com API IDs

### ISSUE THREAD

**Issue by kuroshio** [PokemonTCG/pokemon-tcg-api#91]
I noticed that [Scryfall](https://scryfall.com/)'s API for MTG includes a `tcgplayer_id` field which allows the users of their API to map the MTG card or set to the product on tcgplayer.com.

I was wondering if you would be open to including the tcgplayer.com ID for cards, and possibly even the group ID for sets. To find the price for a card as the API currently stands, I need to maintain a mapping of the pokemon-tcg-api "set codes" to the tcgplayer.com "group IDs" to get a list of products for that set, and then try to match the product based on the card number using the `extendedData` of each product (which doesn't always work as perfectly as one would hope).

If you're open to the idea, I would be happy to help implement this. I can use the existing logic from my personal app to update the JSON files in the [pokemon-tcg-data](https://github.com/PokemonTCG/pokemon-tcg-data) repo for it and submit a PR over there for it.

### Reference Links:

- [Scryfall Card Fields](https://scryfall.com/docs/api/cards)
- [Scryfall Set Fields](https://scryfall.com/docs/api/sets)
- [Example JSON response for a Scryfall card](https://api.scryfall.com/cards/xln/96)
- [Example JSON response for a Scryfall set](https://api.scryfall.com/sets/mmq)

**TL;DR** - Can we add a `tcgPlayerId` field to sets/cards to easily relate them to their respective groups/products on tcgplayer.com?
